src/pages/locationPage.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Card from '../components/Card'
+import location from '../data.json'
+import Header from '../components/Header'
+
+LocationPage.propTypes = {
+  title: PropTypes.string,
+}
+
+export default function LocationPage({ title }) {
+  return (
+    <Grid>
+      <Header>
+        <h2>Best Locations</h2>
+      </Header>
+      <div>
+        {location.map(location => {
+          const { id, attributes } = location
```

Comment:
you could de-structure in line 18 and you can on top de-structure attributes too